parquet-thrift/src/test/resources/parquet/hadoop/thrift/AddressBook.json

```diff
@@ -1,5 +1,6 @@
 {
   "type" : "STRUCT",
+  "isUnion": false,
```

Comment:
actually, now that I think about it -- I need to support old metadata files that don't have this field set.